examples/neuralNetwork-sequence-mouse-gesture-rdp/sketch.js

```diff
@@ -0,0 +1,126 @@
+/*
+ * 👋 Hello! This is an ml5.js example made and shared with ❤️.
+ * Learn more about the ml5.js project: https://ml5js.org/
+ * ml5.js license and Code of Conduct: https://github.com/ml5js/ml5-next-gen/blob/main/LICENSE.md
+ *
+ * This example demonstrates how to train your own mouse gesture classifier
+ * through ml5.neuralNetwork with the sequenceClassificationWithCNN task.
+ */
+
+let model;
+
+let state = "collecting";
+let curShape = "circle";
+let sequence = [];
+let targetLength = 30;
+
+function setup() {
+  let canvas = createCanvas(600, 400);
+  canvas.parent("canvasDiv");
+
+  let options = {
+    inputs: ["x", "y"],
+    outputs: ["label"],
+    task: "sequenceClassificationWithCNN",
+    debug: true,
+    learningRate: 0.005, // smaller learning rate converged better
+  };
+  model = ml5.neuralNetwork(options);
+
+  select("#collCirclesBtn").mouseClicked(collectCircles);
+  select("#collSquaresBtn").mouseClicked(collectSquares);
+  select("#trainBtn").mouseClicked(trainModel);
+}
+
+function draw() {
+  background(220);
+
+  for (let i = 0; i < sequence.length - 1; i++) {
+    line(sequence[i].x, sequence[i].y, sequence[i + 1].x, sequence[i + 1].y);
+  }
+
+  // This sketch uses the RDP line simplification algorithm
+  // to make each inputs to the neural network have the same
+  // number of points.
+  // For more information about RDP, see:
+  // https://www.youtube.com/watch?v=ZCXkvwLxBrA
+
+  let rdp = model.setFixedLength(sequence, targetLength);
```

Comment:
@shiffman @MOQN Wondering if you have some (additional) ideas for how to name this (helper) method, which takes a sequence of indeterminate length and returns a sequence of fixed length - using the RDP algorithm?

Reply:
`simplify`? `rdp`? `resample`?